src/components/widgets/Map.vue

```diff
@@ -797,6 +797,7 @@ const missionFetchProgress = ref(0)
 // Allow fetching missions
 const downloadMissionFromVehicle = async (): Promise<void> => {
```

Comment:
It's not clear to me how the addition of a `clearMapDrawing` function call would "auto sync missions between mission planning and map widget".